cache/in-memory/src/lib.rs

```diff
@@ -462,21 +464,34 @@ impl InMemoryCache {
             .map(|r| Arc::clone(r.value()))
     }
 
-    /// Clears the entire state of the Cache. This is equal to creating a new
-    /// empty Cache.
+    /// Clear the state of the Cache.
+    ///
+    /// This is equal to creating a new empty cache.
```

Comment:
I don't think it will shink memory, but that is probably a technicality.

Reply:
It won't. The underlying table from `hashbrown` states [the memory is kept kept for re-use](https://docs.rs/hashbrown/0.9.1/hashbrown/struct.HashMap.html#method.clear)